src/pybind/mgr/cephadm/module.py

```diff
@@ -2010,8 +1960,9 @@ def zap_device(self, host, path):
             raise OrchestratorError('Zap failed: %s' % '\n'.join(out + err))
         return '\n'.join(out + err)
 
+    @trivial_completion
     def blink_device_light(self, ident_fault, on, locs):
-        @async_map_completion
+        @forall_hosts
```

Comment:
this should run on just one host i think!

Reply:
yes. right now, it's executed for all locs in parallel

Reply:
`forall_hosts` doesn't really work for all "hosts", but maps over all arguments in parallel. Maybe `s/forall_hosts/forall_args/`?